Entities/ServerConfig.cs

```diff
@@ -19,6 +19,235 @@ public class ServerConfig
+				
+		[Column("command_prefix", TypeName = "varchar(255)")]
+		public string CommandPrefix{ get; set; } = "!";
+
+		[Column("command_prefix_alt", TypeName = "varchar(255)")]
+		public string CommandPrefixAlt{ get; set; } = "";
+		
+		[Column("execute_on_edit")]
+		public bool ExecuteOnEdit{ get; set; } = true;
+		
+		[Column("antispam_priority")]
+		public bool AntispamPriority{ get; set; } = false;
+		
+		[Column("antispam_invites")]
+		public bool AntispamInvites{ get; set; } = false;
+		
+		[Column("antispam_invites_ban")]
+		public bool AntispamInvitesBan{ get; set; } = false;
+		
+		[Column("antispam_duplicate")]
+		public bool AntispamDuplicate{ get; set; } = false;
+		
+		[Column("antispam_duplicate_crossserver")]
+		public bool AntispamDuplicateCrossserver{ get; set; } = false;
+		
+		[Column("antispam_duplicate_ban")]
+		public bool AntispamDuplicateBan{ get; set; } = false;
+
+		[Column("antispam_mentions_max")]
+		public Int64 AntispamMentionsMax{ get; set; } = 0;
+		
+		[Column("antispam_mentions_ban")]
+		public bool AntispamMentionsBan{ get; set; } = false;
+		
+		[Column("antispam_mute")]
+		public bool AntispamMute{ get; set; } = false;
+
+		[Column("antispam_mute_duration")]
+		public Int64 AntispamMuteDuration{ get; set; } = 5;
+		
+		[Column("antispam_links_extended")]
+		public bool AntispamLinksExtended{ get; set; } = false;
+		
+		[Column("antispam_links_extended_ban")]
+		public bool AntispamLinksExtendedBan{ get; set; } = false;
+		
+		[Column("antispam_links_standard")]
+		public bool AntispamLinksStandard{ get; set; } = false;
+		
+		[Column("antispam_links_standard_ban")]
+		public bool AntispamStandardStandardBan{ get; set; } = false;
+		
+		[Column("antispam_links_youtube")]
+		public bool AntispamLinksYoutube{ get; set; } = false;
+		
+		[Column("antispam_links_youtube_ban")]
+		public bool AntispamLinksYoutubeBan{ get; set; } = false;
+		
+		[Column("antispam_links_twitch")]
+		public bool AntispamLinksTwitch{ get; set; } = false;
+		
+		[Column("antispam_links_twitch_ban")]
+		public bool AntispamLinksTwitchBan{ get; set; } = false;
+		
+		[Column("antispam_links_hitbox")]
+		public bool AntispamLinksHitbox{ get; set; } = false;
+		
+		[Column("antispam_links_hitbox_ban")]
+		public bool AntispamLinksHitboxBan{ get; set; } = false;
+		
+		[Column("antispam_links_beam")]
+		public bool AntispamLinksBeam{ get; set; } = false;
+		
+		[Column("antispam_links_beam_ban")]
+		public bool AntispamLinksBeamBan{ get; set; } = false;
+		
+		[Column("antispam_links_imgur")]
+		public bool AntispamLinksImgur{ get; set; } = false;
+		
+		[Column("antispam_links_imgur_ban")]
+		public bool AntispamLinksImgurBan{ get; set; } = false;
+
+		[Column("antispam_tolerance")]
+		public Int64 AntispamTolerance{ get; set; } = 7;
+		
+		[Column("antispam_ignore_members")]
+		public bool AntispamIgnoreMembers{ get; set; } = false;
+		
+		[Column("roleid_operator")]
+		public guid RoleIdOperator{ get; set; } = 0;
```

Comment:
Can you please make this `operator_roleid` and `OperatorRoleId`? I'll fix that in the database. For the sake of consistency with "MuteRoleId," etc...